package.json

```diff
@@ -15,7 +15,7 @@
     "public good"
   ],
   "license": "MIT",
-  "name": "KodaDot NFT Marketplace",
+  "name": "kodadot-nft-marketplace",
```

Comment:
```suggestion
  "name": "KodaDot NFT Marketplace",
```

Reply:
wut?

Reply:
It's for solving the eslint error. Seems that the error would only occur after the first commit.
there is a rule for package name 
<img width="748" alt="image" src="https://user-images.githubusercontent.com/31397967/212947622-4c7eb3a5-9d13-46e9-bbba-16240b3ee2cd.png">